ios/Sources/FavoritesFeature/FavoritesFeature.swift

```diff
@@ -19,16 +19,19 @@ public struct FavoriteItem: Equatable, Identifiable {
 public struct FavoritesState: Equatable {
     // TODO: Replace to real models
     public var items: [FavoriteItem]
+    public var isSettingActive: Bool
 
     public init(items: [FavoriteItem] = []) {
         self.items = items
+        self.isSettingActive = false
     }
 }
 
 public enum FavoritesAction {
     case refresh
     case tap(FavoriteItem)
     case favorite(FavoriteItem)
+    case settings(_ isActive: Bool)
```

Comment:
isActive is not necessary 🙏 
```suggestion
    case toogleSettings
```

Reply:
I think unless this action has associated value, the state "isSettingActive" (changed to "isSettingPresented" now) can't  
be operated well and transition works well...  Do you have the other idea to operate this state well?

Reply:
Sorry for the inadequate explanation.
By creating two action for present and dismiss will do. https://github.com/DroidKaigi/conference-app-2021/blob/c37efb66d074f41fe963f3208c5b30850f52d22b/ios/Sources/MediaFeature/MediaFeature.swift#L43
In this case, I think both look fine. Thanks for asking.